app/views/artsy_auth/sessions/new.erb

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Artsy Auth</title>
+  </head>
+  <body style="display: none;">
+    <%= button_to 'Log in via Artsy', '/auth/artsy', form: { id: 'artsy-auth-login-form' } %>
+    <script>
+      document.getElementById('artsy-auth-login-form').submit()
```

Comment:
A form (not displayed) submitted by javascript.

I assume this new page can be displayed in the future and let users choose how to log in, when we support multiple auth methods/providers.

Reply:
I've noticed some implementations render a little text here (something simple like "Authenticating..."). Maybe this would provide a slightly better experience for slow connections/devices?

Reply:
OK now we'll display a simple/unstyled "Authenticating..." fallback for slow clients. I personally don't like flashing content so I only show the text after ~1 second, hoping for regular clients it'll still be seamless. Curious how you think!

- slow clients
![slow-clients](https://user-images.githubusercontent.com/796573/127568591-7863f434-b28d-45f7-83ea-db396ab30549.gif)

- fast clients
![fast-clients](https://user-images.githubusercontent.com/796573/127569136-e630a9ef-d186-4417-944a-d46d3b3023b6.gif)

Reply:
Looks good to me. I'd even give it another second or 2, but up to you.